src/app/components/AccountInfo/DepositModal.tsx

```diff
@@ -86,7 +90,7 @@ class DepositModal extends React.Component<Props> {
 
     return (
       <Modal
-        title="BTC Address"
+        title={blockchainDisplayName[chain] + " Address"}
```

Comment:
Should this be `coinSymbols[chain]` instead to keep it being "BTC Address"? Unless you want it to display "Bitcoin Address" now.

Reply:
Good call, Charlie. Pushing this up right now.